contrib/IECoreArnold/src/IECoreArnold/RendererImplementation.cpp

```diff
@@ -238,7 +238,7 @@ void IECoreArnold::RendererImplementation::camera( const std::string &name, cons
 	AiNodeSetInt( options, "yres", resolution->readable().y );
 
 	const FloatData *pixelAspectRatio = cortexCamera->parametersData()->member<FloatData>( "pixelAspectRatio" );
-	AiNodeSetFlt( options, "aspect_ratio", 1.0f / pixelAspectRatio->readable() ); // arnold is y/x, we're x/y
+	AiNodeSetFlt( options, "pixel_aspect_ratio", 1.0f / pixelAspectRatio->readable() ); // arnold is y/x, we're x/y
```

Comment:
I could have sworn I saw something saying that they'd also changed the interpretation to be x/y - am I wrong about that?

Reply:
Ah - good catch, thanks, will fix.